app/backend/aquifers/migrations/0006_auto_20181003_1820.py

```diff
@@ -7,7 +7,7 @@
 
 
 class Migration(migrations.Migration):
-    atomic = settings.DATABASES.get('default').get('engine') == 'django.db.backends.postgresql'
+    atomic = settings.DATABASES.get('default').get('engine') == 'django.contrib.gis.db.backends.postgis'
```

Comment:
We can drop all this logic, and just have atomic as you've done with the other files.

Reply:
Agreed.. done.